app/helpers/application_helper.rb

```diff
@@ -62,4 +62,9 @@ def content_for_title(title, title_url)
     return title unless title_url
     link_to title, title_url, class: "t-link--black"
   end
+
+  def flash_message(name, msg)
+    return sanitize(msg) if name.end_with? "html"
+    msg
+  end
```

Comment:
Could you add some tests for this in application helper test?

As an aside, for context, to add a link to the blog post in the banner, the message is marked to be safe to render as html if the flash type ends in html. The other flash messages will continue to be escaped.